lib/tasks/compare.rake

```diff
@@ -0,0 +1,24 @@
+require 'compare_links'
+
+namespace :compare do
+  desc """
+  Compare expanded links with links for migration for a schema_name
+  rake 'compare:links[service_manual_guide]'
+  """
+  task :schema_name, [:schema_name] => :environment do |_t, args|
+    schema_name = args[:schema_name]
+    ContentItem.where(schema_name: schema_name).each do |item|
```

Comment:
quibble but this could be thousands of items, so probably better to use `find_each`.